docs/administration/back_office/back_office_elements/reusable_components.md

```diff
@@ -42,6 +42,92 @@ The alert component has the following properties:
 } only %}
 ```
 
+## Details
+
+The details component consists of the following blocks:
+
+- `details_header`
+
+- `details_items`
+
+Variables:
+
+|Name|Type|Values|
+|----|----|-----------|
+|`headline` (optional)|string|if not defined, `details_header` is empty|
+|`headline_items`|array|
+|`view_mode`|string|`vertical`, default set to `''`|
+|`items`|array<hash>|{`label`, `content_raw`, `content`}|
+
+If `headline` is passed, the `table_header` is loaded to `details_header` and then it is possible to pass `headline_items` variable.
+
+`headline` and `headline_items` are variables for `@ibexadesign/ui/component/table/table_header.html.twig`
+
+## Modal
+
+The modal component consist of the following blocks:
+
+```html+twig
+{% block dialog %}
+	{% block content_before %}
```

Comment:
Cosmetic: too big indents. This should not matter in functionality, but doesn't look nice.